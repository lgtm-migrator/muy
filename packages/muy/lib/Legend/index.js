import React from "react"
import Box from "@material-ui/core/Box"

const Legend = props => (
  <Box
    boxSizing="border-box"
    color="inherit"
    component="legend"
    display="table"
    maxWidth="100%"
    p={0}
    whiteSpace="normal"
    {...props}
  />
)

export default Legend
