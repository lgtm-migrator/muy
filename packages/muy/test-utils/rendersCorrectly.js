import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
