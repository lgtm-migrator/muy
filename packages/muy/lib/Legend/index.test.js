import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import Legend from "."

describe("<Legend/> component", () => {
  it("should render correctly with no props", () => {
    rendersCorrectly(<Legend />)
  })
})
