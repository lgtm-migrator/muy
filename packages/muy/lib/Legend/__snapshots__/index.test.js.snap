// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Legend/> component should render correctly with no props 1`] = `
<legend
  className="MuiBox-root MuiBox-root-2 makeStyles-root-1"
/>
`;
